data/app.js

```diff
@@ -0,0 +1,44 @@
+const jsonServer = require('json-server');
+
+const server = jsonServer.create();
+const middlewares = jsonServer.defaults();
+const v1Router = jsonServer.router('v1.json');
+
+const hydraRender = (req, res) => {
+  let { data } = res.locals;
+
+  if (req.originalUrl.startsWith('/v1/playlistScreenRegion')) {
```

Comment:
Is this end-point in the API design?

Reply:
Its a hack for handling the relations. This is the url that is in the mock api json. I'll add a comment about it